docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,539 @@
+    ```
+    ret = noti_ex_item_set_id(item_handle, "new_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Style:
+
+  The style attribute sets the color, padding, and geometry for an item. You can also specify the background image and color.
+
+  - Color
+
+    Color information can be defined as a `notification_ex color` handle with alpha, red, green, and blue value.
+    ```
+    noti_ex_color_create(noti_ex_color_h *handle, unsigned char a, unsigned char r, unsigned char g, unsigned char b);
+    ```
+
+  - Padding
+
+    Padding information can be defined as a `notification_ex padding` handle with left, right, top, and bottom values.
+    ```
+    noti_ex_padding_create(noti_ex_padding_h *handle, int left, int top, int right, int bottom);
+    ```
+
+  - Geometry
+
+    Geometry information can be defined as a `notification_ex geometry` handle with x, y, width, and height values.
+    ```
+    noti_ex_geometry_create(noti_ex_geometry_h *handle, int x, int y, int w, int h);
+    ```
+
+  To set the style attribute, use `noti_ex_item_set_style()`:
+    ```
+    noti_ex_style_create(noti_ex_style_h *handle, noti_ex_color_h color, noti_ex_padding_h padding, noti_ex_geometry_h geometry);
+
+    ret = noti_ex_item_set_style(item_handle, style);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Sound, Vibration and LED:
+
+  To set the sound and vibration attribute for a notification, use `noti_ex_item_set_sound_path()` and `noti_ex_item_set_vibration_path()`:
+    ```
+    ret = noti_ex_item_set_sound_path(item_handle, "sound_path");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_set_vibration_path(item_handle, "vibration_path");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+  To set the LED attribute for a notification, use `noti_ex_led_info_create()` and `noti_ex_led_info_set_on_period()`:
+
+  `noti_ex_led_info_create()` sets the LED operation and the second parameter defines the LED color.
+  `noti_ex_led_info_set_on_period()` and `noti_ex_led_info_set_off_period()` sets the time period when the LED is switched on and off.
+
+    ```
+    noti_ex_color_create(&color, 0, 0, 0, 0);
+
+    noti_ex_led_info_create(&led_info, color);
+    noti_ex_led_info_set_on_period(led_info, 5);
+    noti_ex_led_info_set_off_period(led_info, 5);
+    ```
+
+- Channel:
+
+  Each notification is displayed by a channel.
+    ```
+    ret = noti_ex_item_set_channel(item_handle, "channel");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Receiver Group:
+
+  If you specify a receiver group that you want to receive, all the viewers in that group receive a notification.
+    ```
+    ret = noti_ex_item_add_receiver(item_handle, NOTI_EX_RECEIVER_GROUP_INDICATOR);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+  Followiing are the five viewer types:
```

Comment:
Following .
Also I think viewer types can be listed, instead of code block.
What do you think?

Reply:
done